egs/europarl_st/SRT/lcma_srt/decode.py

```
@@ -0,0 +1,1538 @@
+            enable_st=params.enable_st,
+        )
+        moe_weights_asr = None
+        moe_weights_st = None
+
+    asr_hyps = []
+    st_hyps = []
+    if params.decoding_method == "fast_beam_search":
+        hyp_tokens = fast_beam_search_one_best(
+            model=model,
+            decoding_graph=decoding_graph,
+            encoder_out=encoder_out,
```

Ruff (0.15.20)

[error] Undefined name `encoder_out`

(F821)